usaspending_api/reporting/management/commands/populate_reporting_agency_overview.py

```diff
@@ -0,0 +1,407 @@
+        fiscal_year INTEGER,
+        fiscal_quarter INTEGER,
+        quarter_format_flag BOOLEAN
+    );
+
+    CREATE TEMPORARY TABLE {TempTableName.VALID_FILE_D.value} (
+        toptier_code TEXT,
+        award_id INTEGER,
+        is_fpds BOOLEAN,
+        fiscal_year INTEGER,
+        fiscal_period INTEGER
+    );
+
+    CREATE TEMPORARY TABLE {TempTableName.REPORTING_OVERVIEW.value} (
+        fiscal_period INTEGER,
+        fiscal_year INTEGER,
+        toptier_code TEXT,
+        total_dollars_obligated_gtas NUMERIC(23, 2),
+        total_budgetary_resources NUMERIC(23, 2),
+        total_diff_approp_ocpa_obligated_amounts NUMERIC(23, 2)
+    );
+
+    CREATE TEMPORARY TABLE {TempTableName.AWARD_COUNTS.value} (
+        toptier_code TEXT,
+        fiscal_year INTEGER,
+        fiscal_period INTEGER,
+        unlinked_procurement_c_awards INTEGER,
+        unlinked_assistance_c_awards INTEGER,
+        unlinked_d1_awards INTEGER,
+        unlinked_d2_awards INTEGER,
+        linked_procurement_awards INTEGER,
+        linked_assistance_awards INTEGER
+    );
+
+    ----- Any Indexes to increase performance go below here -----
+"""
+
+TEMP_TABLE_CONTENTS = {
+    TempTableName.VALID_FILE_C: """
+        SELECT DISTINCT
+            ta.toptier_code,
+            faba.award_id,
+            faba.distinct_award_key,
+            CASE WHEN faba.piid IS NOT NULL THEN true ELSE false END AS is_fpds,
+            sa.reporting_fiscal_year AS fiscal_year,
+            sa.reporting_fiscal_quarter as fiscal_quarter,
+            sa.reporting_fiscal_period AS fiscal_period,
+            sa.quarter_format_flag
+        FROM
+            financial_accounts_by_awards AS faba
+        INNER JOIN
+            submission_attributes AS sa USING (submission_id)
+        INNER JOIN
+            dabs_submission_window_schedule AS dsws ON (
+                sa.submission_window_id = dsws.id
+                AND dsws.submission_reveal_date <= now()
+            )
+        INNER JOIN
+            treasury_appropriation_account AS taa ON (taa.treasury_account_identifier = faba.treasury_account_id)
+        INNER JOIN
+            toptier_agency AS ta ON (taa.funding_toptier_agency_id = ta.toptier_agency_id)
+        WHERE
+            faba.transaction_obligated_amount IS NOT NULL
+            AND sa.reporting_fiscal_year >= 2017
+    """,
+    TempTableName.QUARTERLY_LOOKUP: f"""
+        SELECT DISTINCT
+            toptier_code,
+            fiscal_year,
+            fiscal_quarter,
+            quarter_format_flag
+        FROM
+            {TempTableName.VALID_FILE_C.value}
+        WHERE
+            quarter_format_flag = true
+    """,
+    TempTableName.VALID_FILE_D: f"""
+        SELECT DISTINCT
+            fa.toptier_code,
+            awards.id AS award_id,
+            awards.is_fpds,
+            transactions.fiscal_year,
+            CASE
+                WHEN ql.quarter_format_flag = true THEN transactions.fiscal_quarter * 3
```

Comment:
Would be good to add a comment here to describe why the equation is used instead of period (for the poor-man's quarter rollup)

Reply:
bump

Reply:
Really don't know why it isn't showing outdated

Reply:
🤷  doesn't matter. I saw the changes after I refreshed